<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="iDADMINLabel">
       <property name="text">
        <string>ID ADMIN</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="iDADMINLineEdit">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="uSERNAMELabel">
       <property name="text">
        <string>USERNAME</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="uSERNAMELineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pASSWORDLabel">
       <property name="text">
        <string>PASSWORD</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="pASSWORDLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="nAMALabel">
       <property name="text">
        <string>NAMA</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="nAMALineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="eMAILLabel">
       <property name="text">
        <string>EMAIL</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="eMAILLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="rOLELabel">
       <property name="text">
        <string>ROLE</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="rOLEComboBox">
       <item>
        <property name="text">
         <string>admin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>staff</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="tANGGALDIBUATLabel">
       <property name="text">
        <string>TANGGAL DIBUAT</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="tANGGALDIBUATLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>SIMPAN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>HAPUS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>BERSIH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>UBAH</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="EditCari"/>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>id_admin</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>username</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>password</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>nama_lengkap</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>email</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>role</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>tanggal_dibuat</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
